app/build.gradle

```diff
@@ -131,10 +132,12 @@ dependencies {
     implementation "androidx.navigation:navigation-fragment-ktx:2.3.0"
     implementation "androidx.navigation:navigation-ui-ktx:2.3.0"
 
-    // Koin for Android
-    implementation "org.koin:koin-android:2.0.1"
-    implementation "org.koin:koin-androidx-scope:2.0.1"
-    implementation "org.koin:koin-androidx-viewmodel:2.0.1"
+    // Dagger-Hilt
+    implementation "com.google.dagger:hilt-android:2.29.1-alpha"
```

Comment:
nevím jestli je dobrý nápad, vzhledem k tomu proč migrujeme, používat alpha verze nové knihovny

Reply:
Jakou jinou DI bys použil? Čistý Dagger je pain, za mě Hilt ok.

Reply:
Už jsem psal na Slacku že to je alfa. Ale mám tuhle verzi v práci v produkci a bez problémů.